src/advanced/configuration-updater.md

```diff
@@ -432,7 +432,7 @@ and its signature are computed automatically.
 
 #### Parameters
 
-- **config_hash_vote_for**: Hash  
+- **hash**: Hash  
```

Comment:
Is `hash` passed in the request body? If so, please specify, like in the "Submit Configuration Proposal" section.